.github/workflows/ql-for-ql-build.yml

```diff
@@ -5,13 +5,6 @@ on:
     branches: [main]
   pull_request:
     branches: [main]
-    paths:
```

Comment:
why remove these paths?

Reply:
I wanted to trigger QL-for-QL on this PR. 
The simplest approach was just to always run QL-for-QL.